src/Microsoft.DotNet.ApiSymbolExtensions/Filtering/AttributeSymbolFilter.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.DotNet.ApiSymbolExtensions
+{
+    public class AttributeSymbolFilter : ISymbolFilter
```

Comment:
As this class is public, please add an xml comment similar to the CompositeFilter.

Reply:
updated